### ISSUE TITLE
Scrape each job separately

### ISSUE THREAD

**Issue by Maros112358** [burningalchemist/sql_exporter#383]
**Is your feature request related to a problem? Please describe.**
I monitor multiple databases. Majority of them respond in ~30 seconds. One database can take up to 10 minutes to respond. To wait for the slow database, I have to scrape the sql_exporter and wait 10 minutes, which is limiting factor when monitoring faster databases.

**Describe the solution you'd like**
I would like to have a query parameter `job=<job_name>`, with which I can choose which job to scrape. For example, in the situation above I would have config like this
```yaml
jobs:
  - job_name: quick_job
    collectors: [some_collector]
    static_configs:
        - targets:
            fast_database_1: 'connection_string_1'
            fast_database_2: 'connection_string_2'
  - job_name: slow_job
    collectors: [another_collector]
    static_configs:
        - targets:
            slow_database: 'connection_string_slow'
```
To query fast databases, I would go to `/metrics?job=quick_job`. To query the slow database, I would goto `/metrics?job=slow_job`.

**Describe alternatives you've considered**
Having two separate sql_exporters (two pods or binaries) side by side, which would require two configuration files, and two different ports in k8s, and so on. This would also make adding new jobs painful process.
I also considered using cached collector, but caching will not help me when the database takes long time to respond.

**Comment by burningalchemist** [comment]
Hey @Maros112358 it's a duplicate of #178. I'll already made a prototype, and it'll be available in `v0.14`. I could make it a part of the upcoming `v0.13` but it's too many features for a single release. 👍

**Comment by Maros112358** [comment]
Hello @burningalchemist, thank you for quick response. I haven't noticed the issue. Closing this
